packages/platform-sdk-intl/src/datetime.ts

```diff
@@ -354,6 +356,22 @@ export class DateTime {
 		return this.#instance.valueOf();
 	}
 
+	public toDate(): Date {
+		return this.#instance.toDate();
+	}
+
+	public startOf(unit: QUnitType): DateTime {
```

Comment:
The `startOf`, `from` and `fromNow` methods should be above the `toObject` method.